### ISSUE TITLE
Recommended temporary/interim solution to antivirus program blocking Bisq 0.9.1

### ISSUE THREAD

**Issue by RoamingPhil** [bisq-network/bisq#2208]
Hi, I know and am experiencing antivirus program (Webroot) blocking Bisq from running. Even if I set Bisq as a whitelisted item and temporarily turn off Webroot when starting Bisq, Bisq will quit shortly after I turn on Webroot.

Since Bisq is needed to run all the time even when waiting for trades, this would mean I have to run my computer (Windows 10) naked without antivirus protection, all the time, and hence not desirable.

I read that the Bisq team is trying to fix the problem. But is there a temporary, interim solution while we wait for the fix?

**Comment by ghost** [comment]
In the long run, using a dedicated linux computer is the right thing.
(a 100$ second-hand PC does a wonderful job).

**Comment by ManfredKarrer** [comment]
@RoamingPhil AV software can be a pain and I don't see much chances that we can fix the issues they cause by false positives soon. One thing though is that we want to add Windows code signing and maybe that helps with some AV software.
I really recommend to use a didicated Linux (or OSX) computer for any crypto currency related activities. Windows is terribly unsafe and AV software does not fix a failed security model.

**Comment by RoamingPhil** [comment]
Thanks for the quick reply.
I agree running dedicated linux computer might be a safer solution.  However, "dedicated" also means not convenient or flexible if I have only one laptop and am traveling all the time.
This problem is even worse for me since Bisq has to be running all the time for the offer to be seen by other Bisq users, not just at the setting up time and trading time.
Is this AV killing Bisq problem only related to the recent updates, i.e., v0.9.0 and v0.9.1?  If yes, I can try to run an old version, i.e., v0.8.0, if it is available for downloading.

**Comment by ManfredKarrer** [comment]
No its not related to update. Try using another AV software (AVG for example is known to not create false positives with Bisq).

**Comment by devinbileck** [comment]
I agree it may not always be practical or feasible to run a dedicated Linux or macOS machine just for Bisq. An alternative is to setup a VPS which can be online all the time. However, this could come with other concerns.

Is Webroot informing you that it is blocking Bisq (e.g. a popup)? Is it quarantining any of Bisq's files?
When you attempted to "whitelist" Bisq, did you add the application directory to an exception list or something?

**Comment by RoamingPhil** [comment]
Virtual machines may be an option.  I may look at later.
I recall Webroot initially flagged (a pop up) the bisq during installation.  I then temporarily disabled Webroot and reinstall, but found it then blocked the bisq.exe file (a pop up) during startup.
I then added the bisq.exe file to the whitelist.  After this, I could start bisq.exe, but only halfway until after I entered the password before it quit automatically.
If I disable Webroot, I can start bisq without a problem.  But shortly after re-enabling webroot, bisq quits.
I will try to add the full application folder to the whitelist to see if it fixed.
Thanks for the suggestion.
